Sources/App/Controllers/Models/Product.swift

```diff
@@ -6,12 +6,57 @@
 //
 
 import Vapor
+import Fluent
 
-struct Product: Content {
-    var id: Int
+final class Product: Model, Content {
+
+    static let schema = "products"
+
+    @ID
+    var id: UUID?
+
+    @Field(key: "name")
     var name: String
+
+    @Field(key: "description")
     var description: String
+
+    @Field(key: "imageURL")
     var imageURL: String
+
+    @Field(key: "price")
     var price: String
+
+    @Field(key: "quantity")
     var quantity: Int
```

Comment:
Can we rename this variable to `stockQuantity` to distinguish from the `Order.quantity`